OracleSOA/dockerfiles/12.2.1.2/Dockerfile.quickstart

```diff
@@ -0,0 +1,40 @@
+# Pull base image
+# ---------------
+FROM oracle/serverjre:8
```

Comment:
Yes, it's based in WebLogic version. 
With reorder do you mean to have an initial domain once we start the container? (i.e. add the 'createAndStartEmptyDomain.sh') How will the base domain looks like? SOA Suite has at least 3 main componets: SOA, BPM and OSB, the empty domain will have only SOA? Why not keep an 'intermediate' layer with the product installed and have additional images based on that one, one for each main component?

I think we can probably reuse the infrastructure layer to create a SOA "generic" image.